packages/client/src/api/forum.ts

```diff
@@ -0,0 +1,35 @@
+import {apiForum} from './request';
+import {
+	TCreateForumRequest, TCreateForumResponse, TForumListRequest,
+	TForumListResponse, TCreateTopicRequest, TCreateTopicResponse,
+	TTopicListRequest, TTopicListResponse, TCreateMessageRequest,
+	TCreateMessageResponse, TMessageListRequest, TMessageListResponse,
+} from './typingForum';
+import {FORUM_URL} from './constants';
+
+export const forumdAPI = {
+
+	createForum: (data: TCreateForumRequest) => (
+		apiForum.post<TCreateForumResponse>(FORUM_URL, data)
```

Comment:
```suggestion
		apiForum.post<TCreateForumResponse>(FORUM_URL, {
			action: 'нужный_тип',
			data: data,
		}),
```
Писал об этом в прошлый раз. Думаю, это было бы очень удобно. Типы экшенов для форума можно вынести в enum.